src/loaders/svg/tvgSvgLoaderCommon.h

```diff
@@ -233,6 +234,10 @@ struct SvgMaskNode
     bool userSpace;
 };
 
+struct SvgCssStyleNode
+{
```

Comment:
empty for now, attribs like `title`, `type` can be introduced, although I have a problem with finding out how to use the `title` attrib. that's why I assume for now, that only one style element is used in the svg file...